crates/gen-guest-teavm-java/tests/codegen.rs

```diff
@@ -4,6 +4,8 @@ use std::path::Path;
 use std::process::Command;
 
 macro_rules! codegen_test {
+    (worlds_with_types $name:tt $test:tt) => {};
```

Comment:
cc @dicej I added a sample `export_types` binding above but this test fails with path lookup errors that I fear are beyond me at this time.

Reply:
Thanks for letting me know -- I'll take a look today or tomorrow.